notebooks/214-grammar-correction/README.md

```diff
@@ -0,0 +1,31 @@
+# Grammatical Error Correction with OpenVINO
+
+
+Grammatical Error Correction (GEC) is the task of correcting different kinds of errors in text such as spelling, punctuation, grammatical, and word choice errors. 
+GEC is typically formulated as a sentence correction task. A GEC system takes a potentially erroneous sentence as input and is expected to transform it into its corrected version. See the example given below: 
+
+| Input (Erroneous)                                         | Output (Corrected)                                       |
+| --------------------------------------------------------- | -------------------------------------------------------- |
+|She see Tom is catched by policeman in park at last night. | She saw Tom caught by a policeman in the park last night.|
```

Comment:
```suggestion
| I like to rides my bicycle. | I like to ride my bicycle. |
```

Reply:
If you have other examples that you think are better, please feel free to suggest a better one. @eaidova